### PR TITLE
Kured Sandbox announcement

### PR DESCRIPTION
**Pull request by dholbach**
config: allow raw html

Fixes: kubereboot/kured#635

Signed-off-by: Daniel Holbach <daniel@weave.works>

### CODE REVIEW

File: content/en/blog/2022-12-14-kured-sandbox-announcement.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+  entry as well
+- created a [Twitter account](https://twitter.com/kubereboot) and a
+  [mailing list](https://lists.cncf.io/g/cncf-kured-dev) and
+- moved all of our other infrastructure over to CNCF resources, e.g.
+  [our Slack channel](https://cloud-native.slack.com/archives/kured)
+  and more
+
+[1.12.0](https://github.com/kubereboot/kured/releases/tag/1.12.0)
+was the latest release, which was tested against Kubernetes 1.26 and
+introduced the build tools GoReleaser, Cosign and Syft.
```
Comment (ckotzbauer) [inline content/en/blog/2022-12-14-kured-sandbox-announcement.md:68]: We introduced GoReleaser, Cosign and Syft already in 1.11

**Review by ckotzbauer — COMMENTED** [review]
Very good blog-post, thanks for that. Only one small comment from me.

**Comment by dholbach** [comment]
Thanks for the quote @monadic!

**Review by ckotzbauer — APPROVED** [review]

**Review by kingdonb — APPROVED** [review]
👀 LGTM 👍

**Review by jackfrancis — APPROVED** [review]
/lgtm

**Pull request merged**